src/main/kotlin/ch/komunumo/server/business/configuration/control/ConfigurationService.kt

```diff
@@ -17,14 +17,17 @@
  */
 package ch.komunumo.server.business.configuration.control
 
+import mu.KotlinLogging
+import java.io.File
 import java.io.FileInputStream
 import java.nio.file.Paths
 import java.util.Properties
 
 object ConfigurationService {
 
-    private const val configurationFilename = "komunumo.cfg"
-
+    private const val defaultConfigurationFilename = "komunumo.cfg"
+    private val logger = KotlinLogging.logger {}
+    private const val envVariableName = "komunumoconfig"
```

Comment:
Thank you very much for your PR! I would prefer to have environment variable names in UPPERCASE. That's more common.

Reply:
And I think `envVariableName` is not a good variable name. ;-)
How about `environmentConfigurationFilename`? That would match with `defaultConfigurationFilename`.